packages/typespec-autorest/src/utils.ts

```diff
@@ -16,8 +16,8 @@ import {
   Service,
   Type,
 } from "@typespec/compiler";
+import { capitalize } from "@typespec/compiler/casing";
 import { getOperationId } from "@typespec/openapi";
-import { pascalCase } from "change-case";
```

Comment:
I think this also means this can be removed as a dependency(also in typespec-autorest-cannonical where it shouldn't even be there)

Reply:
removed.